injected/entry-points/integration.js

```diff
@@ -28,7 +28,61 @@ function generateConfig() {
             domain: topLevelUrl.hostname,
             isBroken: false,
             allowlisted: false,
-            enabledFeatures: ['fingerprintingCanvas', 'fingerprintingScreenSize', 'navigatorInterface', 'cookie'],
+            enabledFeatures: ['apiManipulation', 'fingerprintingCanvas', 'fingerprintingScreenSize', 'navigatorInterface', 'cookie'],
+        },
+        featureSettings: {
```

Comment:
This is only for tests, right? Is it necessary to keep the default settings in this file?

Reply:
No, I was going to remove after you'd reviewed :). This is just for the ability to manually run it easily (I'm going to suggest we add this as a feature in future for development that we can take a known config and run with it across page navs).